Realm/RLMObjectSchema.mm

```diff
@@ -105,7 +91,7 @@ + (instancetype)schemaForObjectClass:(Class)objectClass {
     Class cls = objectClass;
     Class superClass = class_getSuperclass(cls);
     NSArray *props = @[];
-    while (superClass != RLMObjectBase.class) {
+    while (superClass && superClass != RLMObjectBase.class) {
```

Comment:
Needed so we can get test schemas that throw exceptions (and `NSObject` in Cocoa has properties)